go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/gophercloud/gophercloud
 
+go 1.14
```

Comment:
whoops - actually, I just noticed that Travis is flagging an error and I think it's due to this. Can you remove this? 

We can probably get rid of the Go 1.11 tests altogether, but I will do that in a separate PR when I get some time. Until then, removing this will be the fastest way.

Reply:
Sorry , I forget delete it. This is auto added by IDE.